StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/ElementDocumentationBase.cs

```diff
@@ -235,28 +235,28 @@ private void HandleDeclaration(SyntaxNodeAnalysisContext context, StyleCopSettin
                     .Where(x => !string.Equals(x.GetName()?.ToString(), XmlCommentHelper.IncludeXmlTag, StringComparison.Ordinal))
                 : documentation.Content.GetXmlElements(this.matchElementName);
 
-            if (!matchingXmlElements.Any())
+            var isInheritingDocumentation = documentation.Content
+                .Any(x => string.Equals(x.GetName()?.ToString(), XmlCommentHelper.IncludeXmlTag, StringComparison.Ordinal));
```

Comment:
❔ Why not this?

```suggestion
            var isInheritingDocumentation =
                documentation.Content.GetFirstXmlElement(XmlCommentHelper.IncludeXmlTag) is object;
```

Reply:
Inspired from the code above:
https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/ad20436460c519798c7d48e6aef0b3f4f606e96a/StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/ElementDocumentationBase.cs#L235